bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,259 @@ class CoffeeLintBear:
+        :param: spaces_after_colon:
+            Determines the number of space after colon when
+            ``spaces_before_and_after_colon`` is `` True``.
+        :param: end_file_with_a_single_newline:
+            Checks if the file ends with a single newline.
+        :param: tab_width:
+            Number of spaces for indentation.
+        :param: no_tabs
+            Forbids tabs in indentation.
+        :param: newlines_after_classes:
+            Checks if there is number equal ``number_of_lines_after_classes`` of
+            lines after classes definition.
+        :param: number_of_lines_after_classes:
+            Determines the number of lines that seperate the a class definition
+            and the rest of the code.
+        :param: no_backticks:
+            Prevents some JavaScript elements like ``eval`` to affect
+            coffeescript.
+        :param: no_implicit_braces:
+            Prohibits implicit braces when declaring object literals.
+
+            Example: If ``no_implicit_braces = True`` then
+            ```
+            1:2, 3:4
+            ``` is prohibited whereas
+            ```
+            {1:2, 3:4}
+            ``` is accepted.
+        :param: no_function_call_without_parentheses:
+            prohibits implicit parentheses on function calls.
+        :param: no_interpolation_in_single_quotes:
+            Prohibits string interpolation in a single quoted string.
+
+            Example: If ``no_interpolation_in_single_quotes = True`` then
+            ```
+            f = '#{bar}'
+            ``` is prohibited whereas
+            ```
+            f = "#{bar}"
+            ``` is accepted.
+        :param: no_stand_alone_at:
+            Checks if stand alone at ``@`` are in use.
+        :param: no_throwing_strings:
+           Prohibits throwing string literals or interpolation.
+        :param: no_trailing_semicolons:
+           Prohibits trailing semicolons since they are not useful.
+
+           Example: If ``no_trailing_semicolon = True``
+           ```
+           x = '1234'; console.log(x)
+           ``` here the semicolon is meaningfull.
+           ```
+           alert('end of line');
+           ``` the semicolon is useless.
+        :param: no_trailing_whitespaces:
+            Checks if they are trailing whitespace in the code.
+        :param: apply_no_trailing_whitespaces_in_comments:
+            Determines if ``no_trailing_whitespace`` rule should be applied to
+            comments.
+        :param: apply_no_trailing_whitespaces_in_empty_line:
+            Determines if ``no_trailing_whitespace`` rule should be applied to
+            empty lines.
+        :param: no_unecessary_double_quotes:
+            Prohibits double quotes unless string interpolation is used or the
+            string contains single quotes.
+        :param: use_english_operator:
+            Determines if ``and``, ``or``, ``is`` and ``isnt`` should be use
```

Comment:
should be ``used`` 
also, i dont know about the language, but is it ``isnt``? or is it not ``is not``?